README.md

```diff
@@ -40,33 +81,157 @@ We request that you cite our following paper (new version coming soon) when you
 }
 ```
 
-## Getting Started and Documentation
-
-For more information, see our [getting started page](https://illixr.github.io/ILLIXR/getting_started/).
 
-## Acknowledgements
+## Getting Started and Documentation
 
-Muhammad Huzaifa led the development of ILLIXR in [Sarita Adve’s research group](http://rsim.cs.illinois.edu/) at the University of Illinois at Urbana-Champaign. Other major contributors include Rishi Desai, Samuel Grayson, Xutao Jiang, Ying Jing, Jae Lee, Fang Lu, Joseph Ravichandran, Finn Sinclair, Henghzhi Yuan, Jeffrey Zhang.
+For more information, see our [Getting Started page][50].
 
-ILLIXR came together after many consultations with researchers and practitioners in many domains: audio, graphics, optics, robotics, signal processing, and extended reality systems. We are deeply grateful for all of these discussions and specifically to the following: Wei Cu, Aleksandra Faust, Liang Gao, Matt Horsnell, Amit Jindal, Steve LaValle, Steve Lovegrove, Andrew Maimone, Vegard &#216;ye, Martin Persson, Archontis Politis, Eric Shaffer, Paris Smaragdis, Sachin Talathi, and Chris Widdowson.
 
-Our OpenXR implementation is derived from [Monado](https://monado.dev). We are particularly thankful to Jakob Bornecrantz and Ryan Pavlik.
+## Acknowledgements
 
-The development of ILLIXR was supported by the Applications Driving Architectures (ADA) Research Center, a JUMP Center co-sponsored by SRC and DARPA, the Center for Future Architectures Research (C-FAR), one of the six centers of STARnet, a Semiconductor Research Corporation program sponsored by MARCO and DARPA, and by a Google Faculty Research Award. The development of ILLIXR was also aided by generous hardware and software donations from ARM and NVIDIA. Facebook Reality Labs provided the [OpenEDS Semantic Segmentation Dataset](https://research.fb.com/programs/openeds-challenge/).
+Muhammad Huzaifa led the development of ILLIXR in [Sarita Adve’s research group][9]
+    at the University of Illinois at Urbana-Champaign.
+Other major contributors include
+    Rishi Desai,
+    Samuel Grayson,
+    Xutao Jiang,
+    Ying Jing,
+    Jae Lee,
+    Fang Lu,
+    Joseph Ravichandran,
+    Finn Sinclair,
+    Henghzhi Yuan,
+    and
+    Jeffrey Zhang.
```

Comment:
this is not up to date with the author list of the current ILLIXR paper version.
Missing Yihan Pang, Boyuan Tian

Reply:
Hows does the [CONTRIBUTORS](https://github.com/ILLIXR/ILLIXR/blob/master/CONTRIBUTORS) list look? (@mhuzai)

Reply:
`README.md` is more general, so let's add Yihan's and Boyuan's names to it.
For `CONTRIBUTORS`, let's stick to the protocol described in `CONTRIBUTING.md`. We can add their names when they merge in their PRs (which is going to happen soon anyway from what I understand).